internal/scanner/scanner.rl

```diff
@@ -32,7 +32,8 @@ import "github.com/influxdata/flux/internal/token"
 	string_lit = '"' ( unicode_value | byte_value )* :> '"';
 
 	regex_escaped_char = "\\" ( "/" | "\\");
-	regex_lit = "/" ( any_count_line | byte_value | regex_escaped_char )+ "/";
+	regex_unicode_value = (any_count_line - ("\\" | "/")) | regex_escaped_char;
+	regex_lit = "/" ( regex_unicode_value | byte_value )+ :> "/";
```

Comment:
It actually may not be necessary.

Basically, `:>` is a way of modifying how ragel prioritizes which branch. Normally, it takes whatever branch makes the token the longest so if you have a `/` as an available pattern, the `:> "/"` would force it to end the first time it hits a `/`. This technique works for strings. But, for regex, because it also mixes with the comment symbol, you have `+` instead of `*`. That way you can't have `//` be a regex token because it doesn't have any contents.

The problem with the old implementation is that since regex used `+` and the inner pattern didn't explicitly remove `/`, you could get the following pattern: `///`. This would show up as a regular expression.

It might not be needed anymore though because the inner pattern doesn't match `/` anymore.

Reply:
Removed it to reduce confusion.

Reply:
👍